### PR TITLE
Markup on the form for creating and editing posts

### PR DESCRIPTION
**Pull request by Pablo-Mdz**
---
name: Pull Request Template

about: 

- The markup on the form for creating and editing posts is complete. There aren't many options available (such as colors or tables), but we can always make changes later. Additionally, you can view the changes when editing. For this reason, I made some style adjustments to improve visibility of the changes, even though the final view is not complete.---

-  You have to install < npm i react-quill > on client folder.

- I create an ordered list on topic page and jump the topic card to go directly to topic details. I didn’t delete the topic card, just let me know if you like this path.

From this page: 
Screenshot 2023-04-05 at 10 30 49

Directly  to this: 

Screenshot 2023-04-05 at 10 31 34

Type of Change
      Bug fix (non-breaking change which fixes an issue)
 [x] New feature (non-breaking change which adds functionality)
     Breaking change (fix or feature that would cause existing functionality to not work as expected)
     Refactoring (no new functionality, just code cleanup/refactoring)
     Documentation update
    Testing Steps
    Screenshots (if applicable)
     Checklist
 [x]  I have performed a self-review of my own code
      I have added tests that prove my fix is effective or that my feature works
      I have added any necessary documentation (if appropriate)

### CODE REVIEW

**Comment by Pablo-Mdz** [comment]
Also you have to install < npm i react-quill > on client folder.

File: client/tailwind.config.js [hunk 1]
```diff
@@ -1,8 +1,27 @@
...
+              900: "#1A202C",
+            },
+          },
+        fontFamily: {
+            'pop': ['pop', 'sans-serif']
+        }
+      },
+    },
+    plugins: [],
+  }
```
Comment (antoniosaleme) [inline client/tailwind.config.js:27]: I like it!

**Review by antoniosaleme — APPROVED** [review]

**Comment by Pablo-Mdz** [comment]
AuthContext on posts:

This new feature ensures that only the user who created a post is able to edit it.

**Comment by Pablo-Mdz** [comment]
Screenshot 2023-04-05 at 19 22 21

 Now each post have the name of the author.

**Pull request merged**